containers/unit_tests/TestDualView.hpp

```diff
@@ -379,6 +422,11 @@ void test_dualview_alloc(unsigned int size) {
   ASSERT_TRUE(test.result);
 }
 
+template <typename Scalar, typename Device>
+void test_dualview_copy_construction_and_assignment() {
+  Impl::test_dualview_copy_construction_and_assignment<Scalar, Device>();
```

Comment:
```suggestion
  Impl::test_dualview_copy_construction_and_assignment<Scalar, Device>()();
```
and we get rid of the constructor and replace run_me wiht operator().

Reply:
OK. Made the change.